setup.py

```diff
@@ -67,7 +67,7 @@
         },
     include_package_data = True,  # Will read MANIFEST.in
     install_requires = [
-        'OpenFisca-Core >=35.2.0,<36.0',
+        'OpenFisca-Core >=35.8.0,<36.0',
```

Comment:
Because that's why the CI test was failing : OpenFisca France code was updated for an upgrade in OpenFisca-Core but the dependency was not updated. Nothing to do with Conda. See https://github.com/openfisca/openfisca-france/issues/1883#issuecomment-1245202777